### PR TITLE
Added example02

### PR DESCRIPTION
**Pull request by f1yn**
I merged all of the commits from the [other PR](https://github.com/stasm/innerself/pull/7) into one. I had a re-basing issue on my local/remote master (when tried to squash), so I created a new fork to avoid potential conflicts.

### CODE REVIEW

File: example02/dist.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
+function html([first, ...strings], ...values) {
```
Comment (stasm) [inline example02/dist.js]: Please remove this file.

In https://github.com/stasm/innerself/commit/ac3cbf2b49ac1a08be64a8abb7e5f1d35e3ed0d2 I made it such that examples don't need their own `.gitignore` files, so no need to add one here, by the way.

**Review by stasm — CHANGES_REQUESTED** [review]
Please remove `dist.js`. Otherwise this looks good to me, thanks!

**Comment by flynnham** [comment]
The changes have been made in commit https://github.com/stasm/innerself/pull/10/commits/5449c9ce06ae3d09992d1b61148d1be23b9a3b81 and https://github.com/stasm/innerself/pull/10/commits/2b1efa2a6d4fee0a0e45e08f133c00f4f881eea2

**Review by f1yn — COMMENTED** [review]
See below. I reverted my last two commits, and pushed with the dist removed. https://github.com/stasm/innerself/pull/10/commits/ef163c49b1ac69518ed3ce3d9d0cd688b7c012cf

File: example02/.gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+node_modules
+dist.js
```
Comment (stasm) [inline example02/.gitignore]: In ac3cbf2 I made it such that examples don't need their own .gitignore files, so there's no need to add one here.

Comment (f1yn) [reply example02/.gitignore]: Alright. I'll remove the gitignore file.

**Comment by f1yn** [comment]
It should be fixed now with the latest commit (I reverted and then pushed with the file removed). https://github.com/stasm/innerself/commit/ef163c49b1ac69518ed3ce3d9d0cd688b7c012cf

**Comment by stasm** [comment]
Perfect, thank you a ton!

**Review by stasm — APPROVED** [review]

**Pull request merged**